src/foam/u2/layout/Card.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.layout',
+  name: 'Card',
+  extends: 'foam.u2.borders.CardBorder',
+  documentation: `
+    A card based on a responsive grid system
+  `,
+
+  imports: [
+    'displayWidth?'
+  ],
+
+  css: `
+    ^ {
+      margin: 8px;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'Int',
+      name: 'cols',
```

Comment:
I personally would say "columns" instead of "cols"  I lean towards longer names :)

Possibly even "smallColumns" "xSmallColumns" "xxSmallColumns"

Reply:
Done just appended in column because the enum uses xs, sm, etc.